### PR TITLE
[RM] Harmonizing capitalization - huge pages

### LINKED ISSUE #2682: Harmonize the capitalization of "huge pages" across documents

### PR DESCRIPTION
**Pull request by EditKoselak**
Partially closes #2682

### CODE REVIEW

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ Hardware resources configured as per the Basic profile (B) such that they are on
 <a name="4.2.1.2"></a>
 #### 4.2.1.2 High Performance Profile
 
-The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
+The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter04.md:349]: for consistency
```suggestion
The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
```

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@pgoyal01, @CsatariGergely: please review and let us merge these

**Review by ulikleber — APPROVED** [review]

**Comment by wmk-admin** [comment]
@pgoyal01, can you please check that your comments are done and approve if you can?

**Comment by walterkozlowski** [comment]
@pgoyal01, can you please check this one, and approve or let us know what you still see outstanding?  Thanks

**Comment by pgoyal01** [comment]
@walterkozlowski some places we use capitalised "H" but at least in 2 places lower case is "h" is used:
1. https://github.com/nokia/CNTT/blob/rm-huge-pages/doc/ref_model/chapters/chapter04.md#4212-high-performance-profile
1. Table 5-1 https://github.com/nokia/CNTT/blob/rm-huge-pages/doc/ref_model/chapters/chapter05.md#511-virtual-compute

**Comment by walterkozlowski** [comment]
@pgoyal01, if we agree on "Huge pages" then we can merge this one, and fix in other places as we find them.  It is not a "huge problem" so to speak...

File: doc/ref_model/chapters/chapter04.md (hunk 1 above)
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter04.md:349]:
```suggestion
The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
```

File: doc/ref_model/chapters/chapter05.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The following sections detail the Cloud Infrastructure Software Profile capabili
 | infra.com.cfg.001 | CPU allocation ratio | Value  | Number of virtual cores per physical core. | `i.cap.016` |
 | infra.com.cfg.002 | NUMA alignment | Yes/No | Support of NUMA at the Host OS and virtualisation layers, in addition to hardware. | `e.cap.007` |
 | infra.com.cfg.003 | CPU pinning | Yes/No | Binds a vCPU to a physical core or SMT thread. Configured in OS and virtualisation layers.| `e.cap.006` |
-| infra.com.cfg.004 | Huge Pages | Yes/No | Ability to manage huge pages of memory. Configured in OS and virtualisation layers. | `i.cap.018` |
+| infra.com.cfg.004 | Huge pages | Yes/No | Ability to manage huge pages of memory. Configured in OS and virtualisation layers. | `i.cap.018` |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter05.md:63]:
```suggestion
| infra.com.cfg.004 | Huge pages | Yes/No | Ability to manage Huge pages of memory. Configured in OS and virtualisation layers. | `i.cap.018` |
```

**Comment by pgoyal01** [comment]
@walterkozlowski I have made suggestions for the two places. You can commit those changes. I will approve in the meantime.

**Review by pgoyal01 — APPROVED** [review]
Approve subject to two typo corrections

**Comment by walterkozlowski** [comment]
@pgoyal01, can you please resolve conflicts?  And then I will merge.
I am in TSC planning meeting.

**Comment by pgoyal01** [comment]
@EditKoselak can you please resolve conflicts - it is not allowing me.

File: doc/ref_model/chapters/chapter05.md (hunk 2 above)
Comment (EditKoselak) [reply doc/ref_model/chapters/chapter05.md:63]: Hi @pgoyal01, I've changed the capitalization to huge pages (with lowercase h and p). There are a couple cases where I used different capitalization:
- in the beginning of sentences (Huge page)
- in tables where the other entries started with uppercase (Huge page)
- in titles where title case was used (Huge Page)

**Review by ildikov — APPROVED** [review]

**Pull request merged**